platform/broadcom/docker-syncd-brcm-rpc/Dockerfile.j2

```diff
@@ -9,8 +9,6 @@ debs/{{ deb }}{{' '}}
 {%- endfor -%}
 debs/
 
-RUN dpkg -P syncd
```

Comment:
+# FIXME: Swap SYNCD to SYNCD_RPC after libsaithrift is compatible with SAI v1.0